src/main/java/eu/enmeshed/model/relationships/RelationshipAuditLog.java

```diff
@@ -12,31 +13,20 @@
 @Getter
 @Setter
 @Builder
-public class RelationshipChange {
-
-  private String id;
-
-  private Type type;
-
-  private Status status;
-
-  private RelationshipChangeRequest request;
-
-  private RelationshipChangeResponse response;
-
-  public enum Type {
-    @JsonProperty("Creation")
-    CREATION
-  }
+public class RelationshipAuditLog {
+  private ZonedDateTime createdAt;
+  private String createdBy;
+  private String createdByDevice;
+  private String Reason;
```

Comment:
Reason should be an enum:

- Creation
- AcceptanceOfCreation
- RejectionOfCreation
- RevocationOfCreation
- Termination
- ReactivationRequested
- AcceptanceOfReactivation
- RejectionOfReactivation
- RevocationOfReactivation
- Decomposition

Reply:
Done